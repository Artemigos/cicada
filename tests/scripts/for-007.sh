for x in foo bar baz
    echo $x
    break
done

echo first round
for x in 1 2 3
    for y in a b c
        if [ "$x" = "2" ]
            break
        fi
        echo "$y$x"
    done
done

echo second round
for x in 1 2 3
    for y in a b c
        if [ "$y" = "b" ]
            break
        fi
        echo "$y$x"
    done
done
