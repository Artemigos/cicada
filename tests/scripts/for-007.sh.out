foo
first round
a1
b1
c1
a3
b3
c3
second round
a1
a2
a3
